src/main/js/components/PostTweetUI.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import "../../css/components/PostTweetUI.scss";
+import { maxTweetLength, postTweet } from "../services/twitter-api";
+
+export default class PostTweetUI extends React.Component {
+
+	constructor() {
+		super();
+		this.state = {
+			input: "",
+			output: ""
+		};
+		this.updateMessage = this.updateMessage.bind(this);
+		this.submitTweet = this.submitTweet.bind(this);
+	}
+
+	static get buttonText() {
+		return "Post";
+	}
+
+	static get successMessage() {
+		return "Successfully posted tweet.";
+	}
+
+	static get failureMessage() {
+		return "Could not post tweet.";
+	}
+
+	updateMessage(event) {
+		this.setState({input: event.target.value});
+	}
+
+	submitTweet() {
+		postTweet((err) => {
```

Comment:
if you're worried about readability, is there a way to not pass in the callback inline?